packages/ui/src/components/Loading/Loading.tsx

```diff
@@ -75,17 +76,18 @@ const Loading = ({
             data-h2-border-width="base(6px)"
             data-h2-border-style="base(solid)"
             data-h2-border-color="base(primary transparent primary transparent)"
-            {...(!pause && {
-              animate: {
-                rotate: [0, 360],
-              },
-              transition: {
-                ease: "linear",
-                duration: 1,
-                repeat: Infinity,
-                repeatDelay: 0,
-              },
-            })}
+            {...(!pause &&
+              !shouldReduceMotion && {
```

Comment:
I like this.  If someone has requested reduced motion they probably appreciate having no spinner animation.  I think it's recognizable in context even when not spinning.

Reply:
Sounds good, thanks for responding to that note🙏